app/utils/strip-html-tags.ts

```diff
@@ -0,0 +1,4 @@
+// Strip html tags from string
```

Comment:
This seems like something we shouldn't usually do outside of tests -- maybe it'd make more sense in `tests/helpers/`?